### ISSUE TITLE
Multi-label classification

### ISSUE THREAD

**Issue by ramizdundar** [lucidrains/vit-pytorch#48]
My dataset contains 5 unique labels for instance:

1 0 NaN 1 0

Where 1 means it has that feature. 0 means it doesn’t and NaN means that we have no observation about that feature and it shouldn’t included in loss.

Is it possible to make ViT multi-label for a dataset like this?

**Comment by lucidrains** [comment]
@ramizdundar Hi Ramiz! Yes, just use BCEWithLogits loss instead!

**Comment by alex26-research** [comment]
> @ramizdundar Hi Ramiz! Yes, just use BCEWithLogits loss instead!

I pass image through vit and get using np.argmax class label(794), how can i get class name by class label?

**Comment by ahmadkhan242** [comment]
Hi @lucidrains ,
Please let me know how to vision transformer for multiclass-multilabel classification?
For now, I am using resnet model. 
Input -> [BSx3x8x100] output -> [BSx1x5x181]
Lable -> [BSx5]
I have a total of 181 classes. And for each input 5 labels are being predicted.
I am using Cross-Entropy loss for this multilabel classification.

**Comment by abhigoku10** [comment]
@lucidrains @ramizdundar  can you share the train.py i want to train it on customd dataset /??

**Comment by dalmeidalorraine** [comment]
@ramizdundar Could you please share an example code for multi-label classification using ViT?
